src/test/Square.test.js

```diff
@@ -13,21 +13,14 @@ describe("<Square />", () => {
     expect(img.src).toBe("blank.png");
   });
 
-  /*
-[], [], []
-[], [], []
-[], [], []
-
-  */
-
   it("render a <Square /> with an X only when clicked on and Player's turn is X", () => {
-    const wrapper = mount(<Square currentPlayer={PLAYER.X} />);
+    const wrapper = mount(<Square currentPlayer={PLAYER.X} gameOver={false} />);
     wrapper.find("img").simulate("click");
     const props = wrapper.find("img").props();
     expect(props.src).toBe(SQUARE_CONTENTS.X);
   });
   it("render a <Square /> with an O only when clicked on and Player's turn is O", () => {
-    const wrapper = mount(<Square currentPlayer={PLAYER.O} />);
+    const wrapper = mount(<Square currentPlayer={PLAYER.O} gameOver={false} />);
```

Comment:
let's remove the requirement that you have to pass in the `gameOver` prop to `<Square />`.